llvm/utils/update_cc_test_checks.py

```diff
@@ -34,29 +35,22 @@
 }
 
 
-def get_line2func_list(args, clang_args):
+def get_line2func_list(clang_cmd: Command):
     ret = collections.defaultdict(list)
     # Use clang's JSON AST dump to get the mangled name
-    json_dump_args = [args.clang] + clang_args + ["-fsyntax-only", "-o", "-"]
+    json_dump_args = clang_cmd.args + ["-fsyntax-only", "-o", "/dev/null"]
```

Comment:
I don't understand the change from stdout to /dev/null output. How can you get an output from that command when doing execute_pipeline?

Reply:
This change is not required, it seems JSON is always printed to stdout. This is just to ensure to other output is produced (although probably -fsyntax-only) is sufficient.

Reply:
Ok